### ISSUE TITLE
Server-side data scrubbing does not scrub the error message

### ISSUE THREAD

**Issue by InterstellarStella** [getsentry/sentry#76846]
### Environment

SaaS (https://sentry.io/)

### Steps to Reproduce

1. Make sure to enable server side data scrubbing on the project settings
2. Include a random password in your error's message, for example, `throw new Error('password:dkvnwkaxicnwekfik');`

### Expected Result

The password gets scrubbed.

### Actual Result

The password does not get scrubbed:
![Image](https://github.com/user-attachments/assets/d52b4ca2-6eba-469a-b7ed-8e9aa31f62b8)

The workaround is to use beforeSend and scrub this field on the SDK side, but customers expected that by enabling this setting, the message of the error would be scrubbed too.

### Product Area

Settings - Security & Privacy

**Comment by jjbayer** [comment]
@InterstellarStella thanks, do you still have the JSON of the event? Not sure what fields the JS SDK populates, `exception.value` should get scrubbed but e.g. `logentry.message` does not.

**Comment by InterstellarStella** [comment]
Yes @jjbayer here's the JSON: https://us.sentry.io/api/0/projects/dragonfruit-us/javascript-react/events/e42038f932f845c5a47088bb63c65d71/json/

**Comment by loewenheim** [comment]
In the linked case it really is the `exception.value` containing the password.

**Comment by loewenheim** [comment]
Hi,

this is caused by this ignore list in Relay:
https://github.com/getsentry/relay/blob/25a3b7ae8d4f69e69c8e1b862bd13484d5a8698c/relay-pii/src/convert.rs#L16-L20

We're currently not sure if we should change this, it might break grouping in unforeseen ways. Is this a problem that a user has actually run into?

**Comment by loewenheim** [comment]
Closing this for now due to lack of activity, please feel free to reopen.

**Comment by InterstellarStella** [comment]
@loewenheim Thanks for looking into this, looks like the user's passwords are being filtered now.
